Sources/Commands/SwiftPackageTool.swift

```diff
@@ -30,44 +30,44 @@ import func POSIX.chdir
 extension PackageToolOptions: XcodeprojOptions {}
 
 private enum Mode: Argument, Equatable, CustomStringConvertible {
-    case Init(InitMode)
+    case initPackage
     case doctor
-    case showDependencies(ShowDependenciesMode)
+    case showDependencies
     case fetch
     case update
     case usage
     case version
-    case generateXcodeproj(String?)
-    case dumpPackage(String?)
+    case generateXcodeproj
+    case dumpPackage
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
         case "init", "initialize":
-            self = try .Init(InitMode(pop()))
+            self = .initPackage
         case "doctor":
             self = .doctor
-        case "show-dependencies", "-D":
-            self = try .showDependencies(ShowDependenciesMode(pop()))
+        case "show-dependencies", "show-deps":
+            self = .showDependencies
         case "fetch":
             self = .fetch
         case "update":
             self = .update
         case "help", "usage", "--help", "-h":
             self = .usage
-        case "version":
+        case "version", "--version":
```

Comment:
It does have some semantics that are more commandy than optiony.  For example, I would imagine that `swift package --version fetch` is an error, since `--version` is really a command, and so is fetch, so that would be two commands.  Most other actual options (verbose, chdir, etc) can be passed along with whatever command is provided.  I guess `--help` is another exception, which argues for it being a command (as it is in many multitools).